account_invoice_inter_company/__init__.py

```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+# © 2013-Today Odoo SA
```

Comment:
This module is developed bu odoo SA ?

Reply:
@mourad-ehm , Originally, I used the 'inter_company_rules' module made by Odoo SA for the V8, which I completely refactored.

Reply:
Ok in that case, please drop the `-Today` and put `2017`.

Also in the contributors, maybe link back to the original module. So something like:

```
* Odoo S.A. (`original module <https://a.link.to.the.module.in.github.or.something>`_) 
```